packages/gamut/src/GridForm/types.ts

```diff
@@ -134,3 +134,15 @@ export type GridFormField =
   | GridFormTextAreaField
   | GridFormHiddenField
   | GridFormSweetContainerField;
+
+export type GridFormSectionTitleBaseProps = {
+  title: string;
+  as?: any;
+  layout?: 'center' | 'left';
+};
+
+export type GridFormSectionType = GridFormSectionTitleBaseProps & {
```

Comment:
Nit: this is a "type", no need to include that in the name.

```suggestion
export type GridFormSection = GridFormSectionTitleBaseProps & {
```

Reply:
It goofs with the name of the GridFormSection component. swapping it over to GridFormSectionProps